coffee-backend/src/main/kotlin/chat/letscoffee/security/oauth2/user/FacebookOAuth2UserInfo.kt

```diff
@@ -0,0 +1,26 @@
+package chat.letscoffee.security.oauth2.user
+
+class FacebookOAuth2UserInfo(attributes: Map<String?, Any?>?) : OAuth2UserInfo(attributes) {
```

Comment:
Change to `Map<String, String>` or (if possible) make the request serialize to a `data class`, so that we can check the attributes on a class rather than as keys in a map

Reply:
Opps, wrote that before I saw it overrode OAuth2UserInfo, might not be possible then...

Reply:
🌄